cogs/events.py

```diff
@@ -43,6 +43,19 @@ async def on_command_error(self, ctx, error):
             em = discord.Embed(color = discord.Color.red(),
                                description = f"`{str(error.param).partition(':')[0]}` is a required argument!")
             await ctx.send(embed = em)
+                               
+    @commands.Cog.listener('on_command_error')
+    async def log_error(self, ctx, error):
+        if isinstance(error, commands.CommandNotFound):
+            return
+        channel = self.bot.get_channel(728994664631500940)
+        em = discord.Embed(
+            title='Error:',
```

Comment:
Have it say Bot Error so it doesn't conflict with site errors format

Reply:
Alright